webapp/security/helpers.py

```diff
@@ -0,0 +1,72 @@
+from collections import Counter
+
+
+def get_summarized_status(cve, ignored_low_indicators, vulnerable_indicators):
+    """
+    Return the summarized status for a CVE
+    """
+
+    more_packages = False
+    if len(cve["packages"]) > 1:
+        more_packages = True
+
+    for package in cve["packages"]:
+        # Check if all statuses for all packages are the same, excluding DNE and empty data
+        if (len({d["status"] for d in package["statuses"] if d["status"] not in {"DNE", ""}}) == 1):
+            #  todo: set status
```

Comment:
```suggestion
            #  todo: set status
```
do you need this still?